### PR TITLE
Proxy release 2024-05-16

### PR DESCRIPTION
**Pull request by vipvap**
## Proxy release 2024-05-16

  **Please merge this Pull Request using 'Create a merge commit' button**

### CODE REVIEW

**Review by skyzh — APPROVED** [review]
LGTM as I don't think proxy release should be blocked on / affected by storage changes.

**Comment by conradludgate** [comment]
> LGTM as I don't think proxy release should be blocked on / affected by storage changes.

There's no codeowner checks for release PRs ;)
We're not releasing proxy yet this week as we are blocked on a bug fix

**Comment by danieltprice** [comment]
Reviewed for changelog.

**Pull request closed without merging**
